CONTRIBUTING.md

```diff
@@ -1,9 +1,29 @@
-Please see [contribution-guide.org](http://www.contribution-guide.org/) for steps we expect from contributors before submitting an issue or bug report. Be as concrete as possible, include relevant logs, package versions etc.
+# How to submit an issue?
 
-Please check the [Gensim FAQ](https://github.com/RaRe-Technologies/gensim/wiki/Recipes-&-FAQ) page before posting.
+First, please see [contribution-guide.org](http://www.contribution-guide.org/) for the steps we expect from contributors before submitting an issue or bug report. Be as concrete as possible, include relevant logs, package versions etc.
+
+Also, please check the [Gensim FAQ](https://github.com/RaRe-Technologies/gensim/wiki/Recipes-&-FAQ) page before posting.
 
 **The proper place for open-ended questions is the [gensim mailing list](https://groups.google.com/forum/#!forum/gensim).** Github is not the right place for research discussions or feature requests.
 
-For developers: see our [Developer Page](https://github.com/piskvorky/gensim/wiki/Developer-page#code-style) for details on code style, testing and similar.
+# How to add a new feature or create a pull request?
+
+1. <a href="https://github.com/RaRe-Technologies/gensim/fork">Fork the Gensim repository</a>
+2. Clone your fork: `git clone https://github.com/<USERNAME>/gensim.git`
+3. Create a new branch based on `develop`: `git checkout -b my-feature develop`
+4. Implement your changes
+5. Check that everything's OK in your branch:
+   - Check it for PEP8: `tox -e flake8`
+   - Build its documentation (works only for MacOS/Linux): `tox -e docs`
+   - Run unit tests: `tox -e py{version}-{os}`, for example `tox -e py27-linux` or `tox -e py36-win` where
+      - `{version}` is one of `27`, `35`, `36` and
+      - `{os}` is either `win` or `linux`
```

Comment:
No testing for OSX?

Reply:
Probably, `linux` should works fine for macOS (but I can't test it)